Tests/README.md

```diff
@@ -144,12 +166,12 @@ the test fixture is already set up):
 
 There are a few things to know about running these tests under NUnit:
 
-- You must run the -x86 versions of NUnit, because XNA won't work with
+- You must run the **-x86** versions of NUnit, because XNA won't work with
   the 64-bit versions.
 - You must disable shadow copying because having it enabled makes it
   impossible for the ContentManager to find any assets.
-  - GUI: Tools > Settings > Test Loader > Advanced
-  - CLI: /noshadow
+  - ```GUI: Tools > Settings > Test Loader > Advanced```
+  - ```CLI: /noshadow```
```

Comment:
Use 1 quote for single code line like this, not 3 of em, same for a lot of cases above.

Reply:
Actually the entire file is using triple ones, I see XD